### ISSUE TITLE
[Feature Request]: Point to different location of .env file

### ISSUE THREAD

**Issue by georgefromohio** [laravel-idea/plugin#1121]
### Feature Description

I run development locally and execute the code in a docker container for my Laravel application. This sometimes means that I have to point to different .env files or have differently named .env files. However unless Idea has the correct .env file, it keeps having PhpStorm throw out so many errors see [(stack-trace.txt)](https://github.com/user-attachments/files/18136388/stack-trace.txt)! And sometimes cannot resolve where references are.
```
java.nio.file.NoSuchFileException: /users/<path-to>/laravel-app-folder/.env
    ...
    ...
```
My feature request would be that, you provide a means to point to an alternate .env file for cases like this before your helpers and pointers begin working. Thank you!

**Comment by georgefromohio** [comment]
That was insane! Thank you @adelf! This kind of stuff is the extra sauce!

**Comment by adelf** [comment]
Yeah. It's implemented in 9.1. Thank you for posting this.

Strange exception. The file was found, but when the plugin tried to read it, it got an error.
